egs2/fleurs/asr1/local/nlsyms.txt

```diff
@@ -0,0 +1,103 @@
+af_za
```

Comment:
Is it possible to generate automatically? It is no problem to host here, but if possible generating automatically, could be better. You may refer this for creating the id

Reply:
Yes, this is generated automatically as well when the dataset is downloaded in https://github.com/wanchichen/espnet/blob/master/egs2/fleurs/asr1/local/create_dataset.py#L108-L112

Reply:
Sounds cool! Then we do not have to include that in the PR?

Reply:
Yeah the code in the PR already supports it